### DISCUSSION TITLE
What features of GitHub are you most excited to learn about?

### DISCUSSION THREAD

**Discussion by davidgitit** [Drook-tech/Bootcamp#5] [Q&A]
Actions?
GHAS (code scanning, dependably, etc...)?
Projects?
Issues?
Codespaces?
Pull Requests?
Discussions?
Other?

**Comment by dkalmin** [comment]
I really can't wait for the talk on Actions!

**Comment by xavierxmorris** [comment]
Codespaces with copilot!

**Comment by wicksipedia** [comment]
Codespaces
GHAS

**Comment by Gurenato** [comment]
Actions, GHAS, Codespaces, Projects (missed this) :;)

**Comment by Saraquin** [comment]
Actions, GHAS & projects.

**Comment by william-liebenberg** [comment]
Codespaces, GHAS, Projects

**Comment by Rajeshcn02** [comment]
Codespaces, GitHub Action, GHAS

**Comment by s-aqua** [comment]
Actions mostly, but everything really.

**Comment by TristanLee9608** [comment]
Codespaces, Action, GHAS

**Comment by ramesh2051** [comment]
GitHub Actions, GHAS & Codespaces,
